src/menu_nanox.c

```diff
@@ -0,0 +1,52 @@
+#include <os_io_seproxyhal.h> // todo put it somehow inside the guarded section
+#if defined(TARGET_NANOX)
+
+#include "menu.h"
+#include "getVersion.h"
+#include "glyphs.h"
+
+#define LINES(...) { __VA_ARGS__ }
```

Comment:
Please add comment that this is temporary macro for improving UX_FLOW declarations readability

Reply:
done